packages/e2e-tests/specs/keyboard-navigable-blocks.test.js

```diff
@@ -0,0 +1,165 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	createNewPost,
+	insertBlock,
+} from '@wordpress/e2e-test-utils';
+
+describe( 'Order of block keyboard navigation', () => {
+	beforeEach( async () => {
+		await createNewPost();
+	} );
+
+	it( 'permits tabbing through paragraph blocks in the expected order', async () => {
+		const paragraphBlocks = [ 'Paragraph 0', 'Paragraph 1', 'Paragraph 2' ];
+
+		const navigateToContentEditorTop = async () => {
+			// Use 'Ctrl+`' to return to the top of the editor
```

Comment:
Noting, that this can be coded using the existing helper:
```js
await pressKeyWithModifier( 'ctrl', '`' );
await pressKeyWithModifier( 'ctrl', '`' );
```
the same way as here:
https://github.com/WordPress/gutenberg/blob/master/packages/e2e-tests/specs/block-hierarchy-navigation.test.js#L64